use/wordcount-example.md

```diff
@@ -0,0 +1,413 @@
+
+### Creating the Pipeline
+
+The first step in creating a Beam pipeline is to create a `PipelineOptions object`. This object lets us set various options for our pipeline, such as the pipeline runner that will execute our pipeline and any runner-specific configuration required by the chosedn runner. In this example we set these options programmatically, but more often command-line arguments are used to set `PipelineOptions`. 
+
+In our example, we selected `DataflowRunner` for our Beam runner. This means our pipeline will be executed in the cloud using Google Cloud's Dataflow Service. There are other runners you can specify for executing your pipeline, such as `FlinkRunner` or `SparkRunner`. If you omit specifying a runner, your pipeline will be executed locally. 
+
+```
+ PipelineOptions options = PipelineOptionsFactory.create();
+
+    // In order to run your pipeline, you need to make following runner specific changes:
+    //
+    // CHANGE 1/3: Select a Beam runner, such as DataflowRunner or FlinkRunner.
+    // CHANGE 2/3: Specify runner-required options.
+    // For DataflowRunner, set project and temp location as follows:
+    //   DataflowPipelineOptions dataflowOptions = options.as(DataflowPipelineOptions.class);
+    //   dataflowOptions.setRunner(DataflowRunner.class);
+    //   dataflowOptions.setProject("SET_YOUR_PROJECT_ID_HERE");
+    //   dataflowOptions.setTempLocation("gs://SET_YOUR_BUCKET_NAME_HERE/AND_TEMP_DIRECTORY");
+    // For FlinkRunner, set the runner as follows. See {@code FlinkPipelineOptions}
+    // for more details.
+    //   options.setRunner(FlinkRunner.class);
+```
+
+The next step is to create a Pipeline object with the options we've just constructed. The Pipeline object builds up the graph of transformations to be executed, associated with that particular pipeline.
+
+```
+Pipeline p = Pipeline.create(options);
+```
+
+### Applying Pipeline Transforms
+
+The Minimal WordCount pipeline contains several transforms to read data into the pipeline, manipulate or otherwise transform the data, and write out the results. Each transform represents an operation in the pipeline.
+
+Each transform takes some kind of input (data or otherwise), and produces some output data. The input and output data is represented by the SDK class `PCollection`. `PCollection` is a special class, provided by the Beam SDK, that you can use to represent a data set of virtually any size, including infinite data sets.
+
+<img src="{{ "/images/wordcount-pipeline.png" | prepend: site.baseurl }}">
+Figure 1: The pipeline data flow.
+
+The Minimal WordCount pipeline contains five transforms:
+
+1. A text file `Read` transform is applied to the Pipeline object itself, and produces a `PCollection` as output. Each element in the output PCollection represents one line of text from the input file.
+
+    ```
+    p.apply(TextIO.Read.from("gs://apache-beam-samples/shakespeare/*"))
```

Comment:
Need to introduce that that the example happens to use input data stored in a publicly accessible Google Cloud Storage bucket ("gs://").